### ISSUE TITLE
[atom plugin] "this" in arrow functions not working properly

### ISSUE THREAD

**Issue by genexliu** [ternjs/tern#730]
sample code:
```javascript
// tested on atom 1.5.4 with atom-ternjs 0.13.2 installed
class a {
  foo() {}
  bar() {
    this.foo();   // jumps to foo
    [].forEach(() => {
      this.foo(); // does not jump to foo
    });
  }
}
```

I pasted the same code to the live demo http://ternjs.net/doc/demo/#ES6 but both of the `this.foo()` calls work on the live demo, so this issue might be atom specific.

**Comment by genexliu** [comment]
I will report this issue to https://github.com/tststs/atom-ternjs instead.

**Comment by marijnh** [comment]
The fix for that (676841d6baa4406) wasn't in the latest release yet. I've released 0.18.0 which should fix this.

**Comment by genexliu** [comment]
Great! Thanks for fast reply!
